src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
@vhmth @paulius005 Great idea for a utility file. The chrome.* namespace is provided in Firefox currently in order to help developers port extensions over from Chrome painlessly. 

From 
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Chrome_incompatibilities :
"As a porting aid, the Firefox implementation of WebExtensions supports chrome and callbacks as well as browser and promises."

On the other hand, Microsoft as usual is going for the lovely land of polyfills, with the need to add additional scripts from them in your extension folder in order to make existing chrome extensions work: https://docs.microsoft.com/en-us/microsoft-edge/extensions/guides/porting-chrome-extensions